crates/weaver_forge/src/lib.rs

```diff
@@ -95,6 +98,37 @@ impl Object for TemplateObject {
     }
 }
 
+/// A params object accessible from the template.
+#[derive(Debug, Clone)]
+struct ParamsObject {
+    params: HashMap<String, Value>,
+}
+
+impl ParamsObject {
+    /// Creates a new params object.
+    pub(crate) fn new(params: HashMap<String, serde_yaml::Value>) -> Self {
+        let mut new_params = HashMap::new();
+        for (key, value) in params {
+            _ = new_params.insert(key, Value::from_serialize(&value));
+        }
+        Self { params: new_params }
+    }
+}
+
+impl Display for ParamsObject {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        f.write_str(&format!("{:#?}", self.params))
```

Comment:
Nit:  I'm not a fan of relying on debug formatting...  Can/should we create a formatter for HashMap and use it here and/or just a helper method?

Reply:
This `Display` implementation is unused. I will remove it in the next commit.